daemon/src/model/cfd.rs

```diff
@@ -999,11 +999,11 @@ impl Cfd {
         Ok(Some(new_state))
     }
 
-    pub fn refund_tx(&self) -> Result<Transaction> {
+    pub fn refund_tx(&self) -> Option<Transaction> {
```

Comment:
Will clean that up later, accidentally committed some changes that were not supposed to be where, `cfd.rs`